api/portainer.go

```diff
@@ -1022,7 +1022,7 @@ type (
 		// User Identifier
 		ID       UserID `json:"Id" example:"1"`
 		Username string `json:"Username" example:"bob"`
-		Password string `json:"Password,omitempty" example:"passwd"`
+		Password string `json:"Password,omitempty"`
```

Comment:
I think we should use [swagignore](https://github.com/swaggo/swag#use-swaggerignore-tag-to-exclude-a-field) option

I wonder if the field should ever be returned in any json response.  So in that case, should this field be skipped?  I think there is a way to tell it to be skipped always even when not empty with `json:"-"`.  We'll probably still need the swagignore as well.

Reply:
yes, it should _never_ be returned in any response - its only for sending to portainer to mutate the password

Reply:
iterating with `yarn build:docs` - if only that was mentioned in our CONTRIBUTING.md :D